### DISCUSSION TITLE
Full title as citekey

### DISCUSSION THREAD

**Discussion by NorthFace1234** [retorquere/zotero-better-bibtex#2665] [Q&A] [answered]
Can I get my citekey to be the full title (every word included) of the book/article, with "_" seperating the word?

**Comment by retorquere** [comment, marked as answer]
Yes: `Title.condense(_)`

**Comment by retorquere** [comment]
Make sure the T is capitalized.

**Reply by NorthFace1234** [reply]
It worked, thanks a lot!
